bauhaus/workflows/secondary/hqMetrics.py

```diff
@@ -16,66 +17,135 @@ def conditionTableType():
 
     def generate(self, pflow, ct):
         outputDict = {}
-        pflow.bundleScript("R/hqrf_pbbamr.R")
+        rscript = "R/hqrf_pbbamr.R"
+        pflow.bundleScript(rscript)
+        controls = "fasta/600bp_Control_c2.fasta"
+        pflow.bundleScript(controls)
```

Comment:
I walked this topic through the office and got Elias to bump an old bugzilla (30971) to the SE Jira board. This is a big deal, which requires rev'ing the run metadata XML schema, etc.
 I'll follow up with the Jira number.